demo-playground/src/main/java/com/google/android/flexbox/validators/IntegerInputValidator.kt

```diff
@@ -14,25 +14,27 @@
  * limitations under the License.
  */
 
-package com.google.android.flexbox.validators;
+package com.google.android.flexbox.validators
 
-import android.text.TextUtils;
+import android.text.TextUtils
 
 /**
  * Validator for the integers.
  */
-public class IntegerInputValidator implements InputValidator {
+class IntegerInputValidator : InputValidator {
 
-    @Override
-    public boolean isValidInput(CharSequence charSequence) {
+    override fun isValidInput(charSequence: CharSequence): Boolean {
```

Comment:
Replace the whole body with `return charSequence.toString().toIntOrNull() != null`